test/apps/jest-preset.js

```diff
@@ -0,0 +1,14 @@
+const path = require( 'path' );
+const base = require( '@automattic/calypso-jest' );
+
+module.exports = {
```

Comment:
This copies a lot of things from `test/apps/client` and `test/apps/packages`. It doesn't strictly need to be separate, but probably useful to be more granular, as the projects are all deployed to different locations.